modules/vault-cluster/variables.tf

```diff
@@ -91,6 +96,12 @@ variable "cluster_extra_tags" {
   default = []
 }
 
+variable "instance_extra_tags" {
```

Comment:
Maybe there's a way to derive this from `cluster_extra_tags`, but I don't have the terraform-foo for it.